lib/gooddata/rest/client.rb

```diff
@@ -197,6 +191,10 @@ def connect
       end
 
       def disconnect
+        if stats_on?
+          puts "API call statistics to server #{@connection.server}"
```

Comment:
Good point. The main reason is that `logger` is more flexible. E.g. we can configure a different output. IMHO we should be consistent and use either `puts` or `logger`.

What do you think @korczis?